server/src/parser/table-reader.ts

```diff
@@ -92,6 +92,23 @@ class LineReader {
     }
   }
 
+  private trimPipes(line: string) {
+    // in case empty first cell, do not remove space after leading pipe
+    if (line.startsWith("| | ")) {
+      line = line.substring(1, line.length);
+    }
+    // remove mandatory leading pipe
+    if (line.startsWith("| ")) {
+      line = line.substring(2, line.length);
+    }
```

Comment:
Hi and thank you for you PR! It is highly appreciated. There is however one small issue in how this is implemented. Most of the language server features (goto definition, symbol highlight, etc.) need accurate location information (i.e. line and column). When the current line is mutated here, the line no longer matches the actual line in the editor and all location is off by one (or two).

I think instead of trimming the pipes we should scan for them and skip them. This way the location information will be correct.

Reply:
Sorry for the late response. You are right I missed that. I will try to solve it differently.

Reply:
I changed it to move the position in case of a leading pipe, so the location is not manipulated any more. What do you think?